examples/vic-gov-au/test/e2e/integration/modules/landingPage/CardCollection.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Landing page card collection
+
+  As a citizen I can view a list of cards
+  
+  @skip
+  Scenario: BE - Create landing page
+    # Need to work out how to create a card collection via YAML
```

Comment:
Because `field_paragraph_auto_listing` is all one serialised field its difficult to create via yaml content. I've left the test here as it can be run by manually adding the config in the BE.